x/stakeibc/keeper/icacallbacks_redemption.go

```diff
@@ -61,7 +61,11 @@ func RedemptionCallback(k Keeper, ctx sdk.Context, packet channeltypes.Packet, a
 
 		// Update the unbonding status to TRANSFERRED
 		// TODO: Update with nondeterministic loop
```

Comment:
thanks for adding this! You can remove this comment now

Reply:
I didn't (intentionally) add this - think it showed up as part of a merge. Do you mind taking a quick look to make sure everything merged properly?

Reply:
Yeah I had added that comment to this branch before the merge. Taking a look now though

Reply:
lgtm! I can actually remove the comment cause I'm gonna push something else up right now

Reply:
sounds good